specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/mfe.json

```diff
@@ -19334,6 +19645,42 @@
       },
       "additionalProperties": false
     },
+    "OnlineInferenceConfiguration": {
+      "description": "Online inference configuration options.",
+      "type": "object",
+      "properties": {
+        "configurations": {
+          "description": "Additional configurations",
+          "type": "object",
+          "additionalProperties": {
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=11&length=20)

ARM anti-pattern not allowed. Please schematize this object or model it as an array.